distribution/helm/vector-daemonset/templates/podsecuritypolicy.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
-  name: {{ include "vector.fullname" . }}
+  name: {{ include "libvector.fullname" . }}
```

Comment:
It's likely that we'll need/want different PSP's between a daemon/agent and service deployment. Ex: service wouldn't need to mount the `/var/logs` hostpath since it's likely sourcing from other Vectors or an intermediate buffer.

This isn't a _needed_ change, but would tighten security.

Reply:
Yes, this is a PSP from the `vector-daemonset` (aka `vector-agent`) chart. It's fullname thus will be `vector-daemonset` or `my-release-vector-daemonset`. It should be scoped to the `Service` related to the `DaemonSet` (that also does have the `fullname` with its respective chart name).

For the `StatefulSet`, we'll just have another PSP, this is the plan.

Reply:
Ugh, I think I confused myself - @MOZGIII this **is** actually specific to the daemon/agent deployment right?

Reply:
Is it